tests/test_application.py

```diff
@@ -433,9 +444,10 @@ async def callback(u, c):
             await asyncio.sleep(0.05)
             assert app.update_queue.empty()
             assert self.received == 1
-
-            await app.updater.start_polling()
-            await app.stop()
+            try:  # just in case start_polling times out
```

Comment:
does that tend to happen? If so, please state that in the comment. what happens with the tests when `start_polling` times out? does it pass or fail?

Reply:
so far it happened only once, iirc. It does fail the test yes

Reply:
Mh, one time doesn't sound much to me … and what difference does it make if we stop the app before failing the test? 🤔

Reply:
I will revert this and check if I can reproduce again

Reply:
so if `start_polling` times out, then while exiting the `async with`, we end up with the `This application is still running error`. That's why I stop the updater so it can exit cleanly without failing the test.

Reply:
then please use `except: TelegramError` or something like that. If there's something fundamentally wrong with `updater.start_polling` I wouldn't like to just ignore it here …